### PR TITLE
Added some tips about necessary files (.ipynb), R version, and URL

### PR DESCRIPTION
**Pull request by pablobernabeu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +30,10 @@ are installed by default, so you can use either the Jupyter notebook interface o
 the RStudio interface.
 
 This repository also contains an example of a Shiny app.
+
+Last, note that if you have your Binder URL points to a folder, as in
```
Comment (choldgraf) [inline README.md]:
```suggestion
Last, note that if your Binder URL points to a folder, as in 
```

**Comment by choldgraf** [comment]
this looks great! thanks for the improvement :-)

I had one small typo fix in there and then I'm +1 to merge

**Comment by pablobernabeu** [comment]
thank you!

**Comment by choldgraf** [comment]
(you'll need to accept the suggestion though, github isn't letting me make the change on my own)

**Comment by choldgraf** [comment]
Woo, thanks!

**Pull request merged**